.circleci/config.yml

```diff
@@ -0,0 +1,100 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_USER    - login info for docker hub
+# DOCKER_PASS
+#
+version: 2
+jobs:
+  build:
+    docker:
+      - image: docker:stable-git
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
+    working_directory: /dockerflow
+    steps:
+      - checkout
+      - setup_remote_docker
```

Comment:
What are those referring to?

Reply:
Those are built-in «steps» in Circle CI https://circleci.com/docs/2.0/building-docker-images/